tool/codegen/protoc-gen-auth/file.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 type webApiProjectStore interface {
-	Get(ctx context.Context, id string) (*model.Project, error)
+        GetAllProjectRBACRoles(ctx context.Context, id string) ([]*model.ProjectRBACRole, error)
```

Comment:
Do we need this function actually?
Its implementation is quite simple so if possible I think we can remove it and use `Get` + `GetAllRBACRoles` to simplify https://github.com/pipe-cd/pipecd/blob/1b36f7c402b882ff3f5e37ecef86c8936b15b696/pkg/datastore/projectstore.go#L72-L88 interface.


https://github.com/pipe-cd/pipecd/blob/1b36f7c402b882ff3f5e37ecef86c8936b15b696/pkg/datastore/projectstore.go#L81

Reply:
I felt that the caller doesn't want to be conscious of whether the RBAC roles are all or not when I implemented this.
And I thought this interface might prevent being not called `GetAllRBACRole()`.
wdyt?

Reply:
I get your point, and I'm fine with that. 👍